applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -403,7 +407,12 @@ static void stream_started_cb(struct bt_bap_stream *stream)
 	LOG_INF("Stream %p started", stream);
 
 	if (dir == BT_AUDIO_DIR_SOURCE && IS_ENABLED(CONFIG_BT_AUDIO_TX)) {
-		ERR_CHK(bt_le_audio_tx_stream_started(0));
+		struct stream_index stream_idx = {
```

Comment:
This way of initializing the struct is more readable, but we can consider `struct stream_index stream_idx = {0}`

Reply:
We could, but as you say it might be harder to read

Reply:
Will be removed due to comment above